R/boost_tree.R

```diff
@@ -1,33 +1,21 @@
 # Prototype parsnip code for boosted trees
 
-#' General Interface for Boosted Trees
+#' Boosted trees
+#'
+#' @description
+#'
+#' `boost_tree()` defines a model that creates a series of decision trees that form an
+#' ensemble. Each tree depends on the results of previous trees. All trees in
+#' the ensemble are combined into a final prediction.
+#'
+#' There are different ways to fit this model. Information about the available
+#' _engines_ that can be used for fitting at:
```

Comment:
```suggestion
#' There are different ways to fit this model. See the engine-specific pages for more details:
```
Leaving this here as an example and not spamming all pages with the same suggested change